frontend/app/views/offer/subscriber.scala.html

```diff
@@ -0,0 +1,93 @@
+@import model.Faq
+@import com.gu.membership.salesforce.Tier
+@import model.Benefits
+@import views.support.Prices._
+
+@main("Subscribers") {
+    @defining(Benefits.details(Tier.Partner)) { benefits =>
+        <main role="main">
+
+            <div class="page-slice page-slice--slim l-constrained l-side-margins">
+            @fragments.offer.callout("images/offer/subscriptions/audience-1140.jpg", "Pay nothing for 6 months")
+            </div>
+
+            <section class="tone-white l-gutter l-constrained">
+                <div class="page-section page-section--slim">
+                    <div class="page-section__content">
+                        <p class="text-feature">
+                            A unique opportunity for subscribers to experience the partner tier of Guardian membership (worth £15/ month) free for 6 months.
+                        </p>
+                        <a class="action u-no-margin" href="@routes.Joiner.tierList">Join now</a>
+                    </div>
+                </div>
+            </section>
+
+            <section class="tone-white tone-featured-tint l-gutter l-constrained">
+                <div class="page-section page-section--slim">
+                    <div class="page-section__content">
+                        <h2 class="h-feature">What's included</h2>
+                        <ul class="package-benefits grid grid--3up">
+                        @for(benefit <- benefits.list){
+                            <li class="grid__item">
+                                <div class="u-divider-dotted">@fragments.benefits.item(benefit)</div>
```

Comment:
Nitpick but can `u-divider-dotted go on the parent `li` or does that break the layout?